addon/-private/system/store.js

```diff
@@ -820,13 +820,15 @@ Store = Service.extend({
   _scheduleFetch(internalModel, options) {
     if (internalModel._loadingPromise) { return internalModel._loadingPromise; }
 
-    let modelClass = internalModel.type;
-    let resolver = RSVP.defer('Fetching ' + modelClass.modelName + ' with id: ' + internalModel.id);
+    let { id, modelName } = internalModel;
+    let resolver = RSVP.defer(`Fetching ${modelName}' with id: ${id}`);
     let pendingFetchItem = {
       internalModel,
       resolver,
       options
     };
+
+    let modelClass = internalModel.type; // TODO: is this needed?
```

Comment:
no, this can be refactored to `modelName`.  If so, `this._pendingFetch` should be refactored as well as we don't beed it to expecting `modelClass`